### PR TITLE
chore: upgrade to TypeScript 3.8

### PR DESCRIPTION
**Pull request by fson**
Upgrade TypeScript to 3.8.x. Also Prettier had to be updated to 2.0.x ([TS 3.8 support was introduced in Prettier 2.0](https://prettier.io/blog/2020/03/21/2.0.0.html#typescript)).

### CODE REVIEW

File: packages/next-adapter/src/customize/index.ts [hunk 1]
```diff
@@ -44,7 +44,7 @@ export async function runAsync({
   for (const customization of manifest) {
     const enabled = await customization.onEnabledAsync({ projectRoot, force });
     values.push({
-      name: customization.name,
+      title: customization.name,
```
Comment (dsokal) [inline packages/next-adapter/src/customize/index.ts:47]: Has the new version of Typescript allowed you to find this bug? (was this even a bug? I'm just curious). If not, this seems unrelated.

Comment (fson) [reply packages/next-adapter/src/customize/index.ts:47]: Yes, this resulted in a compilation error with TS 3.8. (Not sure why it wasn't an error before.)
